src/test/java/org/codehaus/plexus/components/io/resources/proxy/ProxyFactoryTest.java

```diff
@@ -65,13 +53,7 @@ public void testCreateProxyWithSizeSupplierOverride()
         throws Exception
     {
         final PlexusIoResourceAttributes s = SimpleResourceAttributes.lastResortDummyAttributesForBrokenOS();
-        SizeSupplier ns = new SizeSupplier()
-        {
-            public long getSize()
-            {
-                return 42;
-            }
-        };
+        SizeSupplier ns = () -> 42;
```

Comment:
This can and the rest can be inlined (` ProxyFactory.createProxy( getPomResource(), () -> 42;);`). Or you think having separate variable is more readable? I'm ok in both cases.

Reply:
I didn't think of it, sure it can be inlined, not sure if it will be more readable, but I'm fine with the inline.